src/components/calcite-link/calcite-link.scss

```diff
@@ -13,18 +13,18 @@
 // link base
 :host a,
 :host span {
-  position: relative;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+  @apply flex 
+    relative 
+    items-center 
+    justify-center 
+    rounded-none 
+    border-none 
+    font-inherit 
+    cursor-pointer;
   text-decoration: none;
-  border-radius: 0;
-  border: none;
   line-height: inherit;
   font-size: inherit;
-  font-family: inherit;
   -webkit-appearance: none;
-  cursor: pointer;
   transition: $transition;
```

Comment:
I keep seeing this line, we really need a `transition-1` helper that just applies our default transition

Reply:
@paulcpederson I thought that's what `@apply focus-base` was for?

Reply:
`focus-base` is for focus styles. If you're doing focus styles here, then you could use it, but we use the `$transition` sass var frequently on its own:

https://github.com/Esri/calcite-components/search?q=%22transition%3A+%24transition%22

Reply:
@paulcpederson oh whoops, you're referring to this one right? 
https://github.com/Esri/calcite-base/blob/master/dist/_variables.scss#L2

If so, how about something like this?
```javascript
// tailwind.config.js
".default-transition": {
    "transition-property": "all",
    "transition-duration": "150ms",
    "transition-timing-function": "ease-in-out",
    "transition-delay": "0s"
  }
// .scss
// @apply default-transition
```

Reply:
maybe `transition-default`? Works for me though, would be all for that addition.